fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImplV2Spec.groovy

```diff
@@ -9,16 +9,14 @@ import spock.lang.Specification
 
 import org.asynchttpclient.Response
 
-import java.util.function.Supplier;
+import java.nio.charset.StandardCharsets
+import java.util.function.Supplier
 
 class AsyncDruidWebServiceImplV2Spec extends Specification {
-    def "Make sure X-Druid-Response-Context and status-code are mreged into existing JsonNode"() {
+    def "Make sure X-Druid-Response-Context and status-code are merged into existing JsonNode"() {
         given:
         Response response = Mock(Response)
-        response.getResponseBody() >>
-                """
-                    {"k1":"v1"}
-                """
+        response.getResponseBodyAsStream() >> new ByteArrayInputStream("{\"k1\":\"v1\"}".getBytes(StandardCharsets.UTF_8))
```

Comment:
Rather than escaping the quotes, we should use Strings (instead of GStrings) if we don't need the templating behavior.